### DISCUSSION TITLE
Welcome to vscode-extension-trello-kanban-board Discussions!

### DISCUSSION THREAD

**Discussion by mayank1513** [mayank1513/vscode-extension-trello-kanban-board#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RoCrow** [comment]
Hey! Great extension, thank you. I just have one question; is it possible to have multiple kanban boards in the same workspace (e.g. by naming each board - 'board-1.kanban', 'board-2.kanban', etc.)

I know we can have one per workspace, but for my needs (using .foam notes) I need to have multiple boards with different names all accessible from the same worksapce.

Is it possible? How can I achieve it?

Also, can I colour code the lists? (e.g. with CSS?)

**Reply by mayank1513** [reply]
No, at present these features are not available.

**Comment by mayank1513** [comment]
Please create feature requests for these if you like to have these features in the next versions.

**Reply by RoCrow** [reply]
Thanks, done :)
